solutions/ceoi-19-AmusementPark.mdx

```diff
@@ -0,0 +1,112 @@
+---
+id: ceoi-19-AmusementPark
+source: CEOI 2019
+title: 2019 - Amusement Park
+author: Shijie Ren
+---
+
+[CF Editorial](https://codeforces.com/blog/entry/68748)
+
+<Spoiler title="Hint">
+
+We can observe that for each legal proposal , reverse direction of every edges ,

```

Comment:
Some grammatical errors here. Maybe something like, "We can observe that for each legal proposal, reversing the direction of its edges creates another legal proposal.